conda/recipes/rmm/recipe.yaml

```diff
@@ -58,7 +59,7 @@ requirements:
     - cython >=3.0.0
     - rapids-build-backend >=0.3.0,<0.4.0.dev0
     - librmm =${{ version }}
-    - python
+    - python =${{ py_version }}
```

Comment:
Just to make sure I understand, we're saying that the difference between conda and rattler is that rattler doesn't pin `python` to the current Python version by default (but conda does). Therefore, we need to ask for the Python version we want explicitly.

Reply:
Yep!